### PR TITLE
Add status badges to README.md

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
Tweaked a few words.

- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] Apache v2 license headers are included in each file.
 
[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/master/CHANGELOG.md

### CODE REVIEW

File: sdk/azidentity/README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
-# Azure Identity client library for Go
+# Azure Identity Client Module for Go
 
-The Azure Identity library provides a set of credential types for use with
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/Azure/azure-sdk-for-go/sdk/azidentity)](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity)
+[![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/go/go%20-%20azidentity?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=1846&branchName=master)
+[![Code Coverage](https://img.shields.io/azure-devops/coverage/azure-sdk/public/1846/master)](https://img.shields.io/azure-devops/coverage/azure-sdk/public/1846/master)
+
+The `azidentity` module provides a set of credential types for use with
```
Comment (catalinaperalta) [inline sdk/azidentity/README.md:7]: Since you have this open already could you change the "which" in this sentence to "that" for me please?

**Review by catalinaperalta — APPROVED** [review]
LGTM

File: sdk/azidentity/README.md (hunk 1 above)
Comment (jhendrixMSFT) [reply sdk/azidentity/README.md:7]: done

Comment (catalinaperalta) [reply sdk/azidentity/README.md:7]: Thanks!

**Pull request merged**